### ISSUE TITLE
Error Chapter's

### ISSUE THREAD

**Issue by HondaiNoir** [CarlosEsco/Neko#1189]
### Steps to reproduce

Error Chapter's are Chapters that doesn't appear in the app but only in the Website (Yes it's translated, you might come and say "Probably Not translated"
![Screenshot_20221012-065146_Neko](https://user-images.githubusercontent.com/115594050/195212920-b44428ad-05cd-4d08-8b2e-bdf7fce46f5d.jpg)
![Screenshot_20221012-065132_Neko](https://user-images.githubusercontent.com/115594050/195212933-e9db8961-d039-414f-b93f-d8d77bfeac9a.jpg)

### Expected behavior

Chapters should appear

### Actual behavior

Chapters didn't appear

### Crash logs

None

### Neko version

2.10.15

### Android version

Android 10

### Device

Galaxy A20S

### Other details

None

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to the newest version **[Latest](https://github.com/CarlosEsco/Neko/releases/latest)**.
- [X] I have filled out all of the requested information in this form.

**Comment by CarlosEsco** [comment]
Did you try swipe refreshing the entry?

Shows fine for me 
![Screenshot_20221011-194516.png](https://user-images.githubusercontent.com/2092019/195218236-dd9f3193-f777-402a-a4db-8393fa296ad7.png)

**Comment by CarlosEsco** [comment]
Looking at your screenshot what chapter are you saying is missing?

**Comment by HondaiNoir** [comment]
I'll try I haven't blocked any Scanlators but they doesn't show

**Comment by HondaiNoir** [comment]
I'll try removing it from my library and put it back on

**Comment by HondaiNoir** [comment]
Didn't work on the History/Update it show's the missing chapter but when i
try to clicked it, it shows this

**Comment by HondaiNoir** [comment]
Didn't work, In the History/Update Page it shows the missing Chapters but
when i click on it, it shows this (Had to Fix my Grammar)

**Comment by HondaiNoir** [comment]
I think the cause of this are No Group's users my Community Page has an
Error that doesn't allow me to see and follow user's

**Comment by HondaiNoir** [comment]
Fixed it, apparently i had blocked "No Group" once accidentally and
unblocked it from the site but in the app it still showed up as Blocked
which keeps me from seeing the chapters uploaded by no group users, it was
just a personal issue

**Comment by CarlosEsco** [comment]
glad it was resovled!
